openpype/hooks/pre_new_console_apps.py

```diff
@@ -24,8 +24,11 @@ def execute(self):
         # - on Windows some apps will create new window using its console
         # Set `stdout` and `stderr` to None so new created console does not
         #   have redirected output to DEVNULL in build
-        self.launch_context.kwargs.update({
-            "creationflags": subprocess.CREATE_NEW_CONSOLE,
-            "stdout": None,
-            "stderr": None
-        })
+        for key, value in (
```

Comment:
Have you read this?
```
# Set `stdout` and `stderr` to None so new created console does not
#   have redirected output to DEVNULL in build
```

Did you try to run e.g. Nuke with these changes? Because I think the Nuke console won't print.

Reply:
`LaunchTypes` won't help. I think there is `is_in_tests` function, that can be used for the check.
```python
from openpype.tests.lib import is_in_tests
```

Reply:
Thanks, done.